consensus/hotstuff/votecollector/statemachine.go

```diff
@@ -175,9 +195,14 @@ func (m *VoteCollector) ProcessBlock(proposal *model.Proposal) error {
 			if errors.Is(err, ErrDifferentCollectorState) {
 				continue // concurrent state update by other thread => restart our logic
 			}
+
+			m.log.Info().
+				Hex("block_id", proposal.Block.BlockID[:]).
+				Msg("vote collector status changed from caching to verifying")
+
```

Comment:
Good point.

I moved the log down, but still above `m.processCachedVotes(proposal.Block)`, so that we print the log as soon as the status has changed. 

And we will see logs in this order
```
vote collector status changed from caching to verifying
processing vote...
processing vote...
```